### PR TITLE
Namespace css classes + update class and style props where applicable

### PR DESCRIPTION
**Pull request by krissalvador27**
- namespaced all css files to have `rct-`
- added class and style props where applicable
- there is a breaking change for prop `markerLineClass` -> `markerLineClassName` in PieChart

### CODE REVIEW

File: src/MarkerLineChart.js [hunk 1]
```diff
@@ -225,7 +237,8 @@ export default class MarkerLineChart extends React.Component {
     if (!_.every([x1, x2, y1, y2], _.isFinite)) return null;
     return (
       <line
-        className="marker-line"
+        className={`${getValue(lineClassName, d, i)}`}
```
Comment (joshbDev) [inline src/MarkerLineChart.js]: does this need to be in strings?

File: src/MarkerLineChart.js [hunk 2]
```diff
@@ -254,9 +276,21 @@ export default class MarkerLineChart extends React.Component {
...
         {...{ x1, x2, y1, y2, key, onMouseEnter, onMouseMove, onMouseLeave }}
       />
     );
   };
+
+  render() {
+    const tickType = getTickType(this.props);
+    return (
+      <g className="rct-marker-line-chart">
+        {tickType === "RangeValue"
```
Comment (joshbDev) [inline src/MarkerLineChart.js]: is `RangeValue` something used across multiple files, and if so should this string be abstracted into a constants file?

File: src/RangeRect.js [hunk 3]
```diff
@@ -85,7 +85,7 @@ export default class RangeRect extends React.Component {
       `RangeRect.props.yScale is not a valid d3 scale`
     );
 
-    const className = `chart-range-rect ${this.props.className || ""}`;
+    const className = `${this.props.className || ""}`;
```
Comment (joshbDev) [inline src/RangeRect.js]: can remove the string

File: src/MarkerLineChart.js (hunk 1 above)
Comment (krissalvador27) [reply src/MarkerLineChart.js]: not sure what you mean by this but to clarify `getValue` takes as its first argument a value or accessor function. so if lineClassName is a value (a string) then it's returned, if it's a function then it's the result of that function applied with the rest of the arguments

File: src/MarkerLineChart.js (hunk 2 above)
Comment (krissalvador27) [reply src/MarkerLineChart.js]: on inspection it's a tickType conceptually used only for MarkerLineChart

File: src/RangeRect.js (hunk 3 above)
Comment (krissalvador27) [reply src/RangeRect.js]: good call!

**Pull request merged**